templates/index.html

```diff
@@ -1,15 +1,14 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <title>ANYWAY - משפיעים בכל דרך</title>
+    <title>{{ gettext('ANYWAY - Making a difference in any way') }}</title>
```

Comment:
@esegal I'm still getting this message in English when I load the site. What could be the problem?